ml_metadata/tools/mlmd_bench/config.pbtxt

```diff
@@ -0,0 +1,23 @@
+mlmd_config: {
```

Comment:
normally put the files for testing in 
testdata/

if this is an example, often we put it in 
README.md
g3doc/  or examples/

Reply:
ditto for the output.pbtxt